package.json

```diff
@@ -47,5 +49,11 @@
     "run-sequence": "^1.0.2",
     "script-loader": "^0.6.1",
     "webpack": "^1.8.3"
+  },
+  "dependencies": {
+    "suitcss-components-arrange": "^0.6.2",
+    "suitcss-components-button": "^4.0.0",
+    "suitcss-components-grid": "^2.0.2",
+    "suitcss-utils": "^0.10.3"
```

Comment:
Probably all of these should be in `devDependencies` unless `suitcss` is special?